kafka-client/src/main/java/dev/responsive/db/CassandraClient.java

```diff
@@ -58,6 +60,8 @@
  */
 public class CassandraClient {
 
+  private static final Logger LOG = LoggerFactory.getLogger(CassandraClient.class);
```

Comment:
We seem to be using LOG some places and LOGGER elsewhere. It seems consistent amongst the storage stuff though.

Reply:
I prefer `LOG` because it's three fewer chars for long log lines 😆 I don't really care though if it's inconsistent

Reply:
we need emoji support in the next java. 🪵 .info("hello")